util/travis-deploy-staging.sh

```diff
@@ -25,6 +25,7 @@ if [[ "${APP_PATH}" == "webapp" ]]; then APP_DEPS="${APP_DEPS}|api|shared"; fi
 if [[ "${APP_PATH}" == "revisions/service" ]]; then APP_DEPS="${APP_DEPS}|revisions|shared"; fi
 # Be more conservative: only deploy searchcache when it's directly modified.
 # if [[ "${APP_PATH}" == "api/query/cache/service" ]]; then APP_DEPS="shared|api/query"; fi
+# if [[ "${APP_PATH}" == "api/query/cache/service/app.staging.yaml" ]]; then APP_DEPS="shared|api/query"; fi
```

Comment:
```suggestion
if [[ "${APP_PATH}" == "api/query/cache/service/app.staging.yaml" ]]; then APP_DEPS="api/query"; fi
```

We need to have this otherwise searchcache won't be deployed even if we modify `api/query/cache/...`.